initrd/bin/kexec-select-boot

```diff
@@ -135,7 +135,7 @@ confirm_menu_option() {
 	if [ "$gui_menu" = "y" ]; then
 		whiptail --clear --title "Confirm boot details" \
 			--menu "Confirm the boot details for $name:\n\n$option\n\n" 20 120 8 \
-			-- 'y' "Boot $name" 'd' "Make $name the default" \
+			-- 'd' "Make $name the default" 'y' "Boot $name" \
```

Comment:
I know why you made this change, but I'm thinking of the common case where someone is booting an alternate option in their boot menu (say a previous kernel, a recovery boot), there's a good chance they *don't* want it to be the default.

Reply:
@kylerankin If selection a specific option, then they should be aware of the specific use case they are taking and choose to not be a default in the future. If by mistake they choose to hit the default, thy have plenty of opportunities down the path to cancel their choice (Modify disk? Seal with TPM, confirm proposed LUKS/LVM Volume). 

In the case they already chose a default before, and the file /boot/kexec_key_devices.txt is or it's LVM counterpart is found, and select to boot that specific option. 

If you really want the default to be to boot the default only, the only alternative I see is to check if /boot/kexec_key_devices.txt of its LVM countrpart exist and propose to set a new default first, else present the choice to boot first. What you think?

Reply:
I keep going back and forth on this one because I see reasons for both being the default. Picking sane default options is hard. I'm fine with the change.